style.css

```diff
@@ -359,12 +359,121 @@ flex-basis: 40%;
     background: whitesmoke;
     height: 50px;
     width: 100%;
+    padding-top: 1.2rem;
 }
 .thumbnail1{
     width: 30px;
-    margin-top: 10px;
+    margin-top: 0;
     border-radius: 50%;
+    padding-bottom: 2px;
+    margin-bottom: 5px;
+    margin-right: 0.70rem;
+}
+#button1{
+   background-color: #000;
+   color: white;
+   border-radius: 20px 20px 20px 20px;
+   padding: 8px 20px;
+   font-size: 1rem;  
+   border: none;
+   margin-left: 2.5rem;
+   margin-bottom: 1rem;
+}
+.left-video-bar p{
+    font-size: 12px;
+    margin-top: 3px;
+   
+}
+.btnright img{
+  width: 16px;
+}
+.btnright{
+   
+}
+#btnlike{
+    padding-right: 30px;
+    
+    border-radius: 20px 0 0 20px;
+    
+    padding-left: 5px;
+    margin: 0;
+    float: left;
+}
+#btndislike{
+    padding-right: 30px;
+    
+    border-radius: 0 20px 20px 0;
+    
+    padding-left: 5px;
+    margin-right: .3rem;
+   
+}
+
+#btnshare,#btndownload{
+    padding-right: 30px;
+ 
+   
+    border-radius: 20px 20px 20px 20px;
+    
+    padding-left: 5px;
+   
+    margin-right: .3rem;
+    
+}
+#btnbar{
+   
+  
+    border-radius:50%;
+    
+    padding-left: 5px;
+    padding-right: 5px;
+   
+   
+}
+.video-bar{
+    display: flex;
+    align-items: center;
+    justify-content:flex-start;
+    
+   
+    
+}
+.left-video-bar{
+    display:inline-flex;
+    margin-right: 9rem;
+}
+.subs-image{
+   align-items: center;
+   padding-top: 0;
+}
+
+.right-video-bar{
+    padding-left: 2rem;
+    align-items: center;
+    margin-right: 0;
+    text-align: center;
+}
+.right-video-bar button{
+    border: none;
+    background:darksalmon;
+    display: inline-flex;
+    padding-top: 8px;
+    padding-bottom: 8px;
+   
+}
+.right-video-bar button:hover{
+    background: #6B728E;
+}
+#button1:hover{
+color:whitesmoke;
```

Comment:
Your overall code is good but you need to make some changes as per below.

- Please kindly change the border-radius value from 12px to 16px as it will be an exact match of the user interface.
- change your other info background color as per user interface. please kindly check it.